core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -61,5 +83,20 @@ def main():
         time.sleep(10)
         sys.exit(1)
 
+    
+def update_cgroups() -> None:
```

Comment:
This should run inside `update_startup()`, previous `main()` where BlueOS `tag, commit_number, commit_hash` are available for necessary checks on the change functions.
You could also move the version check to be inside the new `main()` function and outside the `update_startup()`

Reply:
moved into main